web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -125,9 +118,13 @@ a.card {
   border-radius: 50%;
 }
 
+span.material-icons {
+  color: #FF9049;
```

Comment:
There are other places we use icons. It might be better to use a different class to set the color. Maybe and "orange-icon" class.